README.md

```diff
@@ -56,9 +56,16 @@ Generate an admin user:
 
     bundle exec rake db:devseed    
 
-Fire up a server and import some data.
+Fire up a server:
 
     foreman start -f Procfile.dev
+    
+Or, if you already have Elasticsearch and Redis running because the web and sidekiq actions timeout in `Procfile.dev`, you can start foreman like this:
+
+    foreman start -f Procfile_no_es_redis.dev
```

Comment:
I ran into a couple of different/other issues with foreman's interaction with elasticsearch and redis. The only way I could get things to work was to have redis and ES already running. But I'm open to moving that initializer stuff somewhere else. The first line just ensures the DataSource index is created in ES regardless of where it's run. The rest of that file just writes and executes the Ruby code that sets up the persistence models.